### DISCUSSION TITLE
JavaScript Object Protection guys?

### DISCUSSION THREAD

**Discussion by Kalina67** [Kalina67/Repoistala#3] [Q&A] [answered]
What is a JavaScript Object Protection guys?

**Comment by Aref-abt** [comment, marked as answer]
Hello, Sure here is a simple one:

// Returns true if object is sealed
Object.isSealed(object)

// Prevents any changes to an object
Object.freeze(object)

// Returns true if object is frozen
Object.isFrozen(object)

U might not wanna use this for now, cause you just started but nice question!
